app/assets/javascripts/billables.js

```diff
@@ -2,32 +2,36 @@
 
 $(document).ready(function () {
   function countCheckedBoxes() {
-    var selectedEntriesCount = $('.bill_checkbox:checked').length;
+    var selectedEntriesCount = $('.bill-checkbox.unbilled:checked').length;
     var lang = $('body').data('language');
     $('#amount_marked_entries').text(I18n[lang].checked.pre + selectedEntriesCount + I18n[lang].checked.post);
     $('#submit-billable-entries').attr("disabled", selectedEntriesCount === 0);
   }
 
-  $('#submit-billable-entries').click(function (e) {
-    e.preventDefault();
-    $('#billable-entries-form').submit();
-  });
-
-  $('#billable-entries-form').bind('ajax:complete', function() {
-    window.location.reload();
-  });
-
-  $('.bill_checkbox').change(function () {
+  $('.bill-checkbox').change(function () {
     countCheckedBoxes();
   });
 
-  $('.bill-project').click(function (element) {
+
+  $('.select-all').click(function (element) {
     var checked = $(element.target).prop('checked');
     $('input[data-project-id="' + $(element.target).attr('data-project-id') +
+<<<<<<< HEAD
       '"][name^="' + $(element.target).attr('data-entry_type') + '_to_bill[]"').
+=======
+      '"][name^="' + $(element.target).attr('data-entry_type') + '_ids[]"').
+>>>>>>> unbill buttons added for hours and mileages
```

Comment:
Expected an identifier and instead saw '>>>'.<br>Expected an operator and instead saw '>>>'.<br>Expected an operator and instead saw '>'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(' and instead saw 'hours'.<br>Expected ';' and instead saw 'mileages'.<br>'unbill' is not defined.<br>'buttons' is not defined.<br>Too many errors. (60% scanned).